src/System.Collections.NonGeneric/src/System/Collections/ArrayList.cs

```diff
@@ -64,6 +67,9 @@ public ArrayList()
         // initially empty, but will have room for the given number of elements
         // before any reallocations are required.
         // 
+        /// <summary>Initializes a new instance of the <see cref="T:System.Collections.ArrayList" /> class that is empty and has the specified initial capacity.</summary>
```

Comment:
And you can see that Roslyn offers to simplify the name, both in graying out the namespace and with a quick fix:
![image](https://cloud.githubusercontent.com/assets/2642209/13298262/102b8730-db04-11e5-90a8-70fff9d33ea6.png)